src/pages/how-to-contribute/component/index.mdx

```diff
@@ -19,19 +25,622 @@ To contribute a component to Carbon, start by [opening a Github issue](https://g
+-->
+
+### Responsive behavior
+
+<!--
+Include details about how the component should behave in responsive situations.
+-->
+
+### Default selection
+
+<!--
+Cover guidance around default settings and provide rationale. See [Form](https://www.carbondesignsystem.com/components/form/usage/#default-values) and [Radio button](https://www.carbondesignsystem.com/components/radio-button/usage/#default-selection) for examples.
+-->
+
+### Clickable areas
+
+<!--
+Provide details about the clickable areas of the component.
+-->
+
+### Loading
+
+<!--
+If applicable, use this section to cross reference to the specific loading type in the [Loading component](https://www.carbondesignsystem.com/patterns/loading-pattern) and/or the [Loading pattern](https://www.carbondesignsystem.com/patterns/loading-pattern/).
+-->
+
+## Component type name
+
+<!--
+Rename this H2 heading with the first component type name, and include as many H2 sections as you have types.
+
+This section describes each of the component types listed in the Types table in the Overview, and highlights any behavior that is *unique* to the component type. Cover the topics from Universal behaviors above, as needed.
+
+And again, if any behavior is *universal* to the component, cover that in the "Universal behaviors" section above.
+
+For examples of how to lay out this kind of information, see [Dropdown](https://www.carbondesignsystem.com/components/dropdown/usage) and [Modal](https://www.carbondesignsystem.com/components/modal/usage#variations).
+-->
+
+## Modifiers
+
+<!--
+The modifiers section shows different ways of styling the component. Here are some examples:
+
+- adding an icon to a [button](https://www.carbondesignsystem.com/components/button/usage/#variations)
+- adding a button to a [notification](https://www.carbondesignsystem.com/components/notification/usage/#anatomy)
+- styling a dropdown to appear inline with other content
+- using high and low contrast styles for [notifications](https://www.carbondesignsystem.com/components/notification/usage/#high-and-low-contrast)
+- using a "light" variant in a [text input](https://www.carbondesignsystem.com/components/text-input/code)
+-->
+
+## Related
+
+<!--
+Did you reference other components or patterns? List them here.
+
+If necessary, clarify any differences between this component and related components.
+-->
+
+## References
+
+<!--
+Include links to any research, citations, books or articles that you found helpful.
+-->
+
+## Feedback
+
+Help us improve this component by providing feedback, asking questions, and leaving any other comments on [GitHub](https://github.com/carbon-design-system/carbon-website/issues/new?assignees=&labels=feedback&template=feedback.md).
+```
+
+### Style guidance
+
+We're still working on this template. We'd love your help and feedback!
```

Comment:
@connor-leech do we want to add a call to action here?

Reply:
@janchild Just added a link to an issue 👍